services/thermal_mgr/thermal_mgr.c

```diff
@@ -44,18 +44,70 @@ void initThermalSystemManager(lm75bd_config_t *config) {
 error_code_t thermalMgrSendEvent(thermal_mgr_event_t *event) {
   /* Send an event to the thermal manager queue */
 
-  return ERR_CODE_SUCCESS;
+  if (thermalMgrQueueHandle == NULL || event == NULL)
```

Comment:
thermalMgrQueueHandle == NULL would be an Invalid state error since it is not an argument to the function